wagtail/admin/views/home.py

```diff
@@ -329,7 +329,11 @@ def icons():
         )
         combined_icon_markup = ""
         for icon in all_icons:
-            combined_icon_markup += render_to_string(icon).replace("svg", "symbol")
+            combined_icon_markup += (
+                render_to_string(icon)
+                .replace("svg", "symbol")
+                .replace('xmlns="http://www.w3.org/2000/svg"', "")
```

Comment:
I believe this has to be the other way around, otherwise we’d be create attributes ending with `/symbol`, and then not removing them!

```suggestion
                .replace('xmlns="http://www.w3.org/2000/svg"', "")
                .replace("svg", "symbol")
```